framework/src/imgui_manager.cpp

```diff
@@ -37,7 +54,8 @@ namespace gvk
 	    init_info.Queue = mQueue->handle();
 	    init_info.PipelineCache = nullptr; // TODO: Maybe use a pipeline cache?
 
-		const uint32_t magicImguiFactor = 1000;
+		//const uint32_t magicImguiFactor = 1000;
+		const uint32_t magicImguiFactor = 1;
```

Comment:
That's very cool :) I've still got to test this on AMD for safety, but Intel and Nvidia seem to be totally fine.